src/components/LinkCard/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from '@trussworks/react-uswds';
+
+import './index.scss';
+
+type LinkCardProps = {
+  children: React.ReactNode;
+  className?: string;
+  link: string;
+  heading: React.ReactNode | string;
+} & JSX.IntrinsicElements['div']; // what does this mean?
+
+const LinkCard = ({ children, className, link, heading }: LinkCardProps) => {
+  return (
+    <div
+      className={`padding-2 line-height-body-4 link-card-container bg-base-lightest ${className}`}
```

Comment:
Instead of interpolating `className`, please use [classnames](https://jedwatson.github.io/classnames/).

Most times , it makes things easier. This situation is kinda trivial.

The main benefit of using classnames here is that we wont have dangling `undefined` class name if `className` isn't defined in the props.